v1/metrics.yaml

```diff
@@ -429,115 +439,860 @@ paths:
                 x-client-ip: '{{x-client-ip}}'
       x-monitor: false
 
-definitions:
-  listing:
-    description: The result format for listings
-    required:
-      - items
-    properties:
-      items:
-        type: array
-        items:
-          type: string
+  ########################################
+  # Edited Pages
+  ########################################
+  /edited-pages/:
+    get:
+      tags:
+        - Analytics Edited-Pages Metrics API
+      summary: List mediawiki edited-pages metrics related API entry points.
+      description: |
+        This is the root of mediawiki edited-pages metrics data endpoints. If browsing the interactive documentation, see the specifics for each endpoint below.
 
-  pageview-article:
-    properties:
-      items:
-        type: array
-        items:
-          properties:
-            project:
-              type : string
-            access:
-              type : string
-            article:
-              type: string
-            agent:
-              type: string
-            granularity:
-              type: string
-            timestamp:
-              # the hourly timestamp will be stored as YYYYMMDDHH
-              type: string
-            views:
-              type: integer
-              format: int64
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Stable)
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: The queriable sub-items
+          schema:
+            $ref: '#/definitions/listing'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-monitor: false
 
-  pageview-project:
-    properties:
-      items:
-        type: array
-        items:
-          properties:
-            project:
-              type : string
-            access:
-              type : string
-            agent:
-              type: string
-            granularity:
-              type: string
-            timestamp:
-              # the hourly timestamp will be stored as YYYYMMDDHH
-              type: string
-            views:
-              type: integer
-              format: int64
+  /edited-pages/new/{project}/{editor-type}/{page-type}/{granularity}/{start}/{end}:
+    get:
+      tags:
+        - Analytics Edited-Pages Metrics API
+      summary: Get new pages counts for a project.
+      description: |
+        Given a Mediawiki project and a date range, returns a timeserie of its new pages counts. You can filter by editor type (anonymous, group-bot, name-bot, user) or page-type (all, content or non-content). You can choose between daily and monthly granularity as well. The new pages value is computed as follow: [number of created pages] - [number of deleted pages] + [number of restored pages] for the chosen filters.
```

Comment:
[Nitpicking] Could you split the descriptions into multiple lines? It's gonna be ready painful to rid in code editors. We generally try to limit line length to 100 symbols where possible.

Reply:
Done everywhere in file :)